python/pyarrow/table.pxi

```diff
@@ -433,6 +433,22 @@ cdef class ChunkedArray(_PandasConvertible):
 
         return result
 
+    def sort(self, order="ascending"):
+        """
+        Sort the ChunkedArray
```

Comment:
Should we support the fieldname keyword here as well? (and raise an error in case the type of the ChunkedArray is not a struct)

Reply:
The `ChunkedArray` doesn't provide support for accessing fields directly (IE: no `ChunkedArray.field(X)` method) so we would probably have to implement that too if we want to add support for sorting an individual field. Which seems to expand the scope of this change. I propose we make a separate ticket for improving support for Struct data in ChunkedArray

Reply:
Yes, that's indeed a general issue that ChunkedArray can be annoying when working with a type that type-specific methods on the array (that's not only for StructArray)